pkg/interface/publish/src/js/components/lib/comment-item.js

```diff
@@ -66,7 +66,7 @@ export class CommentItem extends Component {
           </div>
           <div className="f9 gray3 pt1">{date}</div>
         </div>
-        <div className="f8 lh-solid mb8 mb2">
+        <div className="f8 lh-solid mb8 mb2 w-50 w-100-m w-100-s center">
```

Comment:
For all of these: `w-100-s` isn't a class we have, you'd want `w-100 w-50-l w-50-xl` or something.